net/happygears/proto/NSG.proto

```diff
@@ -370,13 +370,13 @@ message PollingVariable {
     int64 index = 1;
```

Comment:
please mark NSGPyton.PollingVariable as deprecated so we dont forget to migrate to this copy

Reply:
Marked deprecated https://github.com/happygears/proto/pull/12/files